include/VecCore/Backend/Implementation.h

```diff
@@ -275,6 +275,7 @@ T Blend(const Mask<T> &mask, const T &src1, const T &src2)
 
 // Miscellaneous
 
+// Note that SIMDsizeUpTo() is more general, as the code below is equivalent to SIMDsizeUpTo<VeryLargeInt>()
```

Comment:
Well, fine, let's add doxygen documentation, but this comment is not very informative, in any case.